### PR TITLE
Present technical conclusions from failed OC project

### LINKED ISSUE #4810: Summarize conclusions from LakeFSOutputCommitter

### PR DESCRIPTION
**Pull request by arielshaqed**
Please add more in review!

Closes #4810.

### CODE REVIEW

**Review by lynnro314 — APPROVED** [review]
Thanks!

**Review by talSofer — COMMENTED** [review]
Thanks @arielshaqed for this writeup, it is very helpful!

File: design/open/output-committer-conclusions.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+KV store.  Work on improving lakeFS performance with DynamoDB (some of this
+work is proposed in #4769, #4734, #4712), and also on improving lakeFSFS on
+lakeFS (some of this work is proposed in #4722, #4721, #4676).
+
+### Poor communication with other teams
+
+We assumed multiple non-functional and one functional requirements that did
+not actually exist.  We should have specified these requirements as part of
+the design and communicated them with VE team.
+
```
[the remark this answers is not in the document]
Comment (arielshaqed) [reply design/open/output-committer-conclusions.md:63]: Thanks!  Added...

**Review by arielshaqed — COMMENTED** [review]
Thanks!

Pulling...

**Comment by arielshaqed** [comment]
Pulling by force: this is a docs PR and functional tests are not blockers to pulling it.

**Pull request merged**